pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -155,8 +165,14 @@ impl IndexPart {
             example_metadata.disk_consistent_lsn(),
             example_metadata,
             Default::default(),
+            RuntimeAuxFilePolicy::V1,
         )
     }
+
+    pub(crate) fn last_aux_file_policy(&self) -> RuntimeAuxFilePolicy {
+        self.last_aux_file_policy
+            .unwrap_or(RuntimeAuxFilePolicy::Unspecified)
```

Comment:
I still fail to understand why is this wrapped in an option if there's `RuntimeAuxFilePolicy::Unspecified`?

Reply:
originally this is used for keep forward compatibility; just decided to remove `RuntimeAuxFilePolicy` and now there is only `AuxFilePolicy`. None means unspecified.